recipes/bloaty/all/conanfile.py

```diff
@@ -0,0 +1,81 @@
+import os
+
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.0"
+
+
+class BloatyConan(ConanFile):
+    name = "bloaty"
+    description = """Bloaty McBloatface: a size profiler for binaries"""
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/google/bloaty"
+    license = "Apache-2.0"
+    topics = ("conan", "bloaty", "profiler", "size")
+
+    settings = "os", "arch", "compiler", "build_type"
+
+    requires = (
+        "capstone/4.0.2",
+        "protobuf/3.17.1",
+        "pkgconf/1.7.4",
+        "zlib/1.2.11",
+        "re2/20210601",
+        "abseil/20210324.2",
+    )
+    exports_sources = "CMakeLists.txt", "patches/**"
+    generators = "cmake", "cmake_find_package", "pkg_config"
+
+    _source_subfolder = "source_subfolder"
+
+    def validate(self):
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, "11")
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration("bloaty package requires Linux")
+
+    def package_id(self):
+        del self.info.settings.compiler
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  destination=self._source_subfolder, strip_root=True)
+
+    def _patch_sources(self):
+        for patch in self.conan_data["patches"][self.version]:
+            tools.patch(**patch)
+
+    _cmake = None
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        if not self.settings.compiler.cppstd:
+            self._cmake.definitions["CMAKE_CXX_STANDARD"] = 11
```

Comment:
When compiler.cppstd is not set, you should force C++11 only if the default standard of the compiler is lower than C++11.

Reply:
Forcing CXX_STANDARD will cause issues with Abseil. See comments here: https://github.com/conan-io/conan-center-index/pull/6330

Reply:
Since the current abseil is broken in this sense, this build has to wait until Abseil is fixed.